modules/global-classes/import-export/export-runner.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Elementor\Modules\GlobalClasses\ImportExport;
+
+use Elementor\App\Modules\ImportExport\Runners\Export\Export_Runner_Base;
+use Elementor\Modules\GlobalClasses\Global_Classes_Repository;
+use Elementor\Modules\GlobalClasses\Global_Classes_Sanitizer;
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Export_Runner extends Export_Runner_Base {
+	public static function get_name(): string {
+		return 'global-classes';
+	}
+
+	public function should_export( array $data ) {
+		// Same as the site-settings runner.
+		return (
+			isset( $data['include'] ) &&
+			in_array( 'settings', $data['include'], true )
+		);
```

Comment:
Check with @aviv-elementor

Reply:
Checked already, he said it's fine for now